src/components/inputs/localized-text-input/localized-text-input.js

```diff
@@ -31,82 +31,86 @@ import {
 
 const sequentialId = createSequentialId('localized-text-input-');
 
-class LocalizedInput extends React.Component {
-  static displayName = 'LocalizedInput';
-  static propTypes = {
-    id: PropTypes.string,
-    name: PropTypes.string,
-    autoComplete: PropTypes.string,
-    value: PropTypes.string.isRequired,
-    onChange: requiredIf(PropTypes.func, props => !props.isReadOnly),
-    language: PropTypes.string.isRequired,
-    onBlur: PropTypes.func,
-    onFocus: PropTypes.func,
-    isAutofocussed: PropTypes.bool,
-    isDisabled: PropTypes.bool,
-    isReadOnly: PropTypes.bool,
-    hasError: PropTypes.bool,
-    placeholder: PropTypes.string,
-  };
-  handleChange = event => {
-    // We manipulate the event to add the language to the target.
-    // That way the users of LocalizedTextInput's onChange can read
-    // event.target.language and event.target.value to determine the next value.
-    //
-    // We only need this information for the story, the MC application code will
-    // never need to access the information in such an inconvenient way, as
-    // Formik can deal with a name like "foo.en" and sets the value correctly.
-    // We can't use this as we aren't guaranteed a name in the story as the user
-    // might clear it using the knob, and then we can't parse the language from
-    // the input name anymore.
-    //
-    // eslint-disable-next-line no-param-reassign
-    event.target.language = this.props.language;
-    this.props.onChange(event);
-  };
-  render() {
-    return (
-      <div
-        key={this.props.language}
-        css={css`
-          width: 100%;
-          position: relative;
-          display: flex;
-        `}
+const LocalizedInput = props => {
+  const { onChange } = props;
+  const handleChange = React.useCallback(
+    event => {
+      // We manipulate the event to add the language to the target.
+      // That way the users of LocalizedTextInput's onChange can read
+      // event.target.language and event.target.value to determine the next value.
+      //
+      // We only need this information for the story, the MC application code will
+      // never need to access the information in such an inconvenient way, as
+      // Formik can deal with a name like "foo.en" and sets the value correctly.
+      // We can't use this as we aren't guaranteed a name in the story as the user
+      // might clear it using the knob, and then we can't parse the language from
+      // the input name anymore.
+      //
+      // eslint-disable-next-line no-param-reassign
+      event.target.language = props.language;
+      onChange(event);
+    },
+    [props.language, onChange]
```

Comment:
👆we only need to destructure `onChange`, for whatever reason. I suppose the lint plugin has some bugs.